itest/rust/src/builtin_tests/geometry/vector_test.rs

```diff
@@ -0,0 +1,67 @@
+use std::f32::consts::PI;
+
+use crate::framework::itest;
+
+use godot::prelude::{
+    inner::{InnerVector2, InnerVector3},
+    *,
+};
+use godot::private::class_macros::assert_eq_approx;
```

Comment:
Nitpick, use the non-prelude variants where possible (here `godot::builtin`) and [flat import statements](https://godot-rust.github.io/book/gdext/contribute/conventions.html#code-organization):
```suggestion
use godot::builtin::{Vector2, Vector3};
use godot::builtin::inner::{InnerVector2, InnerVector3};

use godot::private::class_macros::assert_eq_approx;
```